advisor/src/main/kotlin/advisors/OssIndex.kt

```diff
@@ -77,6 +77,8 @@ class OssIndex(name: String, serverUrl: String = OssIndexService.DEFAULT_BASE_UR
             val componentReports = mutableMapOf<String, OssIndexService.ComponentReport>()
 
             components.chunked(BULK_REQUEST_SIZE).forEach { chunk ->
+                logger.debug { "Querying component report (chunk $chunk of $BULK_REQUEST_SIZE)." }
```

Comment:
Was it intended to include all PURLs and the size limit of a chunk into the log output?

```
val chunks = compontents.chunked(BULK_REQUEST_SIZE)
chunks.forEachIndexed ...
    "Querying $chunk.size component reports (chunk "$i+1 of $chunks.size)"
```

Note: I believe it's a batch query so report**s** IMO should use plural form.

Reply:
> Was it intended to include all PURLs and the size limit of a chunk into the log output?

No, that was an oversight. Thanks for finding it.

> Note: I believe it's a batch query so reports IMO should use plural form.

Within a single loop iteration just one report (for multiple pURLs) is queried, so I believe singular to be correct.